phoenix-core/src/main/java/org/apache/phoenix/coprocessor/IndexRebuildRegionScanner.java

```diff
@@ -98,6 +103,9 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(IndexRebuildRegionScanner.class);
 
+    public static final String PHOENIX_LOG_BEYOND_MAX_LOOKBACK_INVALID_ROWS_CONF_KEY =
+        "phoenix.log.beyond.max.lookback.age.invalid.rows";
+    public static final boolean DEFAULT_PHOENIX_LOG_BEYOND_MAX_LOOKBACK_INVALID_ROWS = true;
```

Comment:
Should this be false? I feel like "beyond max lookback" is kind of analogy to debug log. Why don't we make it false?

Reply:
Done